fix-ssh-on-pi.ini_example

```diff
@@ -3,3 +3,4 @@
 #public_key_file="/home/change_me/.ssh/id_ed25519_pi.pub"
 #wifi_file="/home/change_me/.ssh/wpa_supplicant.conf"
 first_boot="firstboot.sh"
+os_variant="full"
```

Comment:
for consistency, value should be 'full' in single quotes

Reply:
Sorry for the delay in replying to this. I like these ideas but am having a little think about it.